engine/src/launcher.cpp

```diff
@@ -102,7 +102,6 @@ void gv::GvsocLauncher::open()
         signal(SIGINT, sigint_handler);
 
         this->engine_thread = new std::thread(&gv::GvsocLauncher::engine_routine, this);
-        pthread_setname_np(this->engine_thread->native_handle(), "engine");
```

Comment:
I need to set this thread name to properly profile multithreading in gvsoc, is there a constant we can put to disable it only for MAC ?